### ISSUE TITLE
Deprecation warnings at run time

### ISSUE THREAD

**Issue by 0d-billie** [0d-billie/beod.co.uk#10]
`bundle exec jekyll serve` produces the below warnings:

```Deprecation Warning: Sass's behavior for declarations that appear after nested
rules will be changing to match the behavior specified by CSS in an upcoming
version. To keep the existing behavior, move the declaration above the nested
rule. To opt into the new behavior, wrap the declaration in `& {}`.

More info: https://sass-lang.com/d/mixed-decls

   ╷
20 │ ┌   @media (prefers-color-scheme: dark) {
21 │ │     background: $projectbox-dark;
22 │ │   }
   │ └─── nested rule
23 │     height: 200px;
   │     ^^^^^^^^^^^^^ declaration
   ╵
    projects.scss 23:3                                  @import
    /home/bod/dev/beod.co.uk/assets/css/main.scss 23:9  root stylesheet
Deprecation Warning: Sass's behavior for declarations that appear after nested
rules will be changing to match the behavior specified by CSS in an upcoming
version. To keep the existing behavior, move the declaration above the nested
rule. To opt into the new behavior, wrap the declaration in `& {}`.

More info: https://sass-lang.com/d/mixed-decls

    ╷
20  │ ┌   @media (prefers-color-scheme: dark) {
21  │ │     background: $projectbox-dark;
22  │ │   }
    │ └─── nested rule
... │
24  │     width: 100%;
    │     ^^^^^^^^^^^ declaration
    ╵
    projects.scss 24:3                                  @import
    /home/bod/dev/beod.co.uk/assets/css/main.scss 23:9  root stylesheet
Deprecation Warning: Sass's behavior for declarations that appear after nested
rules will be changing to match the behavior specified by CSS in an upcoming
version. To keep the existing behavior, move the declaration above the nested
rule. To opt into the new behavior, wrap the declaration in `& {}`.

More info: https://sass-lang.com/d/mixed-decls

    ╷
20  │ ┌   @media (prefers-color-scheme: dark) {
21  │ │     background: $projectbox-dark;
22  │ │   }
    │ └─── nested rule
... │
25  │     overflow: hidden;
    │     ^^^^^^^^^^^^^^^^ declaration
    ╵
    projects.scss 25:3                                  @import
    /home/bod/dev/beod.co.uk/assets/css/main.scss 23:9  root stylesheet
Deprecation Warning: Sass's behavior for declarations that appear after nested
rules will be changing to match the behavior specified by CSS in an upcoming
version. To keep the existing behavior, move the declaration above the nested
rule. To opt into the new behavior, wrap the declaration in `& {}`.

More info: https://sass-lang.com/d/mixed-decls

    ╷
20  │ ┌   @media (prefers-color-scheme: dark) {
21  │ │     background: $projectbox-dark;
22  │ │   }
    │ └─── nested rule
... │
26  │     display: table;
    │     ^^^^^^^^^^^^^^ declaration
    ╵
    projects.scss 26:3                                  @import
    /home/bod/dev/beod.co.uk/assets/css/main.scss 23:9  root stylesheet```

**Comment by 0d-billie** [comment]
Fixed in #12

**Comment by 0d-billie** [comment]
Fixed in #12
